algorithms/exact_change.js

```diff
@@ -0,0 +1,84 @@
+/*
+https://www.freecodecamp.com/challenges/exact-change
+
+Design a cash register drawer function checkCashRegister() that accepts purchase
+ price as the first argument (price), payment as the second argument (cash), and
+  cash-in-drawer (cid) as the third argument.
+
+cid is a 2D array listing available currency.
+
+Return the string "Insufficient Funds" if cash-in-drawer is less than the change
+ due. Return the string "Closed" if cash-in-drawer is equal to the change due.
+
+Otherwise, return change in coin and bills, sorted in highest to lowest order.
+
+
+*/
+
+
+// Create an object which hold the denominations and their values
+var denom = [
+  { name: 'ONE HUNDRED', val: 100.00},
+  { name: 'TWENTY', val: 20.00},
+  { name: 'TEN', val: 10.00},
+  { name: 'FIVE', val: 5.00},
+  { name: 'ONE', val: 1.00},
+  { name: 'QUARTER', val: 0.25},
+  { name: 'DIME', val: 0.10},
+  { name: 'NICKEL', val: 0.05},
+  { name: 'PENNY', val: 0.01}
+];
+
+
+function checkCashRegister(price, cash, cid) {
+  var change = cash - price;
+
+
+  // Transform CID array into drawer object
+  var register = cid.reduce(function(accumulator, current) {
+    accumulator.total += current[1];
+    accumulator[current[0]] = current[1];
+    return accumulator;
+  }, {total: 0});
+
+  // Handle exact change
+  if (register.total === change) {
+    return 'Closed';
+  }
+
+  // Handle obvious insufficent funds
+  if (register.total < change) {
+    return 'Insufficient Funds';
+  }
+
+  // Loop through the denomination array
+  var change_arr = denom.reduce(function(accumulator, current) {
```

Comment:
I'd call the accumulator `change` or `changeAccumulator` to be more specific to the data you're dealing with. I'd also decompose this into a separate function called `getChange()` or something similar.

Also, be careful of maintaining code style consistency: you've camel cased earlier functions and have used `_` in others. Pick one (JS convention is camel case) and make sure you stick to it. :)